plugin/pkg/scheduler/factory/factory_test.go

```diff
@@ -36,6 +36,10 @@ import (
 	latestschedulerapi "k8s.io/kubernetes/plugin/pkg/scheduler/api/latest"
 )
 
+const (
+	SchedulerName = "default-scheduler"
```

Comment:
I do agree with you, this is messy. #14541 is about such problem. I will figure out a way before #14541 has a solution.